l10n_it_fatturapa_out/wizard/__init__.py

```diff
@@ -18,4 +18,7 @@
 #
 ##############################################################################
 
-from . import wizard_export_fatturapa
+try:
```

Comment:
ok but is useless check here, if the  l10n_it_fatturapa module is not resolved, or not?